lib/spot-request-poller.js

```diff
@@ -0,0 +1,116 @@
+const assert = require('assert');
+const {runAWSRequest} = require('./aws-request');
+const Iterate = require('taskcluster-lib-iterate');
+
+const log = require('./log');
+
+const BATCH_SIZE = 100;
+
+class SpotRequestPoller {
+
+  constructor({ec2, regions, runaws = runAWSRequest, spotPollDelay = 25, state}) {
```

Comment:
Where does the default value for spotPollDelay come from? Why 25, in particular?

Reply:
delivered to me in a dream by divine spirits

Reply:
jk https://github.com/taskcluster/ec2-manager/blob/master/config.yml#L5

Reply:
Do you need to specify a default value if it's already been set in config.yml? I think that would already be taken into account by passing cfg.app.spotPollDelay

Reply:
🖖